README.md

```diff
@@ -1,10 +1,21 @@
 # ABB Experimental
 
-[ROS-Industrial][] ABB experimental meta-package.  See the [ROS wiki][] page for more information.  
-
-## Contents
-
-This repo holds source code for all versions.
+[ROS-Industrial][] ABB experimental meta-package.  See the [ROS wiki][] page for more information.
 
 [ROS-Industrial]: http://www.ros.org/wiki/Industrial
 [ROS wiki]: http://ros.org/wiki/abb_experimental
+
+## Build
+This build methods uses:
+ - [wstool](http://wiki.ros.org/wstool) for rosinstall.
+ - [catkin tools](https://catkin-tools.readthedocs.io/en/latest/) for building.
+```
+mkdir -p abb_experimentnal_ws/src
```

Comment:
Shouldn't it be `abb_experimental_ws` instead of `abb_experimentNal_ws`? The typo is repeated in all lines below as well.